core-bundle/assets/controllers/toggle-fieldset-controller.js

```diff
@@ -16,7 +16,7 @@ export default class extends Controller {
             fs.setAttribute(`data-${identifier}-id-value`, id);
             fs.setAttribute(`data-${identifier}-table-value`, table);
             fs.setAttribute(`data-${identifier}-collapsed-class`, 'collapsed');
-            el.setAttribute('data-action', `click->${identifier}#toggle`);
+            el.setAttribute('data-action', `click->${identifier}#toggle keydown->${identifier}#toggleWithKeyboard`);
```

Comment:
shouldn't this work something like that:

```suggestion
            el.setAttribute('data-action', `click->${identifier}#toggle keydown.enter->${identifier}#toggle`);
```